packages/admin-vue3/src/ui/Sidebar.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <TransitionRoot appear :show="open" as="template">
+        <Dialog as="div" v-bind="$attrs" :open="open">
+            <div
+                class="
+                    fixed
+                    top-0
+                    right-0
+                    z-10
+                    w-64
+                    h-full
+                    pt-12
+                    mt-6
+                    bg-white
+                    border-l border-gray-300
+                    shadow
+                "
+            >
+                <aside class="min-h-screen text-center">
+                    <TransitionChild
+                        as="template"
+                        enter="duration-300 ease-out"
+                        enter-from="opacity-0"
+                        enter-to="opacity-100"
+                        leave="duration-200 ease-in"
+                        leave-from="opacity-100"
+                        leave-to="opacity-0"
+                    >
+                        <DialogOverlay class="fixed inset-0" />
+                    </TransitionChild>
+
+                    <TransitionChild
+                        as="template"
+                        enter="duration-300 ease-out"
+                        enter-from="opacity-0 scale-95"
+                        enter-to="opacity-100 scale-100"
+                        leave="duration-200 ease-in"
+                        leave-from="opacity-100 scale-100"
+                        leave-to="opacity-0 scale-95"
+                    >
+                        <div
+                            class="
+                                inline-block
+                                w-full
+                                max-w-md
+                                px-4
+                                overflow-y-scroll
+                                text-left
+                                transition-all
+                                transform
+                            "
+                        >
+                            <button @click="$emit('close', false)">
+                                <svg
+                                    xmlns="http://www.w3.org/2000/svg"
+                                    viewBox="0 0 24 24"
+                                    width="24"
+                                    height="24"
+                                >
+                                    <path fill="none" d="M0 0h24v24H0z" />
+                                    <path
+                                        d="M7.828 11H20v2H7.828l5.364 5.364-1.414 1.414L4 12l7.778-7.778 1.414 1.414z"
+                                    />
+                                </svg>
+                            </button>
```

Comment:
headlessui needs an element to set autofocus, in this case the element is a button. I figured, it could be this arrow:

![Bildschirmfoto 2021-06-25 um 13 45 22](https://user-images.githubusercontent.com/29352871/123420243-9a582280-d5bb-11eb-88cb-b761e6d6ed31.png)